### PR TITLE
Added guide for embedding external resources

### PR DESCRIPTION
**Pull request by Phil23**

### CODE REVIEW

File: source/assets/css/basic.less [hunk 1]
```diff
@@ -1195,6 +1195,8 @@ input#search-query {
 
 .devdocs {
     font-size: 15px;
+    min-height: 800px;
+    min-height: 80vh;
```
Comment (bcremer) [inline source/assets/css/basic.less]: vh?

Looks like a typo to me. @Phil23

Comment (mitelg) [inline source/assets/css/basic.less]: vh is the px! didn't you get the note @bcremer ?

Comment (bcremer) [inline source/assets/css/basic.less]: should have googled that. stupid hipster css alien technologies nowadays *grumpy*

Comment (Phil23) [inline source/assets/css/basic.less]: vh = viewport height (modern browsers)
px = pixel (legacy browser fallback)

;)

**Pull request merged**